app/src/main/java/crux/bphc/cms/activities/MainActivity.kt

```diff
@@ -61,50 +44,46 @@ class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelecte
         _userAccount = UserAccount(this)
         Constants.TOKEN = _userAccount.token
         _realm = Realm.getDefaultInstance()
-        val toggle: ActionBarDrawerToggle = object : ActionBarDrawerToggle(
-                this, drawer_layout, toolbar, R.string.navigation_drawer_open, R.string.navigation_drawer_close) {
-            override fun onDrawerOpened(drawerView: View) {
-                super.onDrawerOpened(drawerView)
-                val view = currentFocus
-                if (view != null) {
-                    val imm = getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
-                    imm.hideSoftInputFromWindow(view.windowToken, 0)
+
+        bottom_nav.setOnNavigationItemSelectedListener { menuItem ->
+            when (menuItem.itemId) {
+                R.id.myCoursesFragment -> {
+                    pushView(MyCoursesFragment.newInstance(), "My Courses", true)
+                    return@setOnNavigationItemSelectedListener true
```

Comment:
Other than for added  clarity, is there any use for specifying `@setOnNavigationItemSelectedListener`? Wouldn't a simple `true` return from the lambda?